lib/resque/heroku-signals.rb

```diff
@@ -6,16 +6,16 @@
   def unregister_signal_handlers
     trap('TERM') do
       trap('TERM') do
-        puts "[resque-heroku] received second term signal, throwing term exception"
+        log_with_severity :info, "[resque-heroku] received second term signal, throwing term exception"
 
         trap('TERM') do
-          puts "[resque-heroku] third or more time receiving TERM, ignoring"
+          log_with_severity :info, "[resque-heroku] [resque-heroku] third or more time receiving TERM, ignoring"
```

Comment:
@pjambet looks like there's a duplicate `[resque-heroku]` here. Could you remove this?